Makefile

```diff
@@ -66,3 +70,7 @@ check-code:
 	yarn tsc-check
 	yarn eslint-check
 	yarn build
+
+format-code:
+	yarn prettier -w .
+	yarn eslint-check --fix
```

Comment:
running `yarn eslint-fix`should automatically run prettier. prettier has been added as a plugin to eslint so that both are done in the same pass. 

I guess you may not have been aware of it but maybe you create this rule for a different reason?

Reply:
Gotcha, I was not aware of that. I will remove it.